source/voice_control/create_wake_word.markdown

```diff
@@ -0,0 +1,105 @@
+- successfully completed the [$13 voice remote for Home Assistant](/voice_control/thirteen-usd-voice-remote/) tutorial
+
+## To create your own wake word
+
+1. Think of a wake word.
+   - A word or short phrase (3-4 syllables) that is not commonly used so that it does not trigger Home Assistant by mistake.
+   - Currently, only wake words in English are supported.
+2. Open the [wake word training environment](https://colab.research.google.com/drive/1q1oe2zOyZp7UsB3jJiQ1IFn8z5YfjwEb?usp=sharing#scrollTo=1cbqBebHXjFD).
+3. In section 1, enter your wake word in the **target_word** field.
+![Enter wake word in target field](/images/assist/wake_word_enter_target_word.png)
+4. In the code section next to the **target_word**, select the play button. The first time this can take up to 30 seconds.
+   - If the play button does not appear, make sure your cursor is placed in the **target_word** field.
+     ![Select play button](/images/assist/wake_word_press_play_button.png)
+   - If it still does not show up, in the top right corner of the document, make sure it says **Connected**.
+     - If it is not connected, select **Connect to a hosted runtime**.
+     ![Connect to hosted runtime](/images/assist/wake_word_connect_to_hosted_runtime.png)
+   - **Result**: The pronunciation of your wake word is being created.
+     - Once it is finished, at the bottom of the section, you see an audio file. Listen to it.
+  
+     ![Listen to demo of your wake word](/images/assist/wake_word_listen_demo.png)
+5. If the word does not sound correct to you:
+   - Follow the instructions in the document to tweak the spelling of the word and press play again.
+   - The word should sound the way you pronounce it.
+6. Once you are satisfied with the result, in the menu on top of the screen, select **Runtime** > **Run all**.
+   - This will take around an hour. Feel free to do something else but make sure to leave the browser tab open.
+   ![Runtime: run all](/images/assist/wake_word_runtime_run_all.png)
+   - **Result**: Once this process is finished, you should have 2 files in your downloads folder:
+     - `.tflite` and `.onnx` files (only `.tflite` is used)
+
+7. Congratulations! You just applied machine learning to create your own wake word model!
+   - The next step is to add it to Home Assistant.
+
+## To add your personal wake word to Home Assistant
+
+1. Make sure you have the [Samba add-on installed](/common-tasks/os/#configuring-access-to-files).
+2. On your computer, access your Home Assistant server via Samba.
+   - Open the `share` folder and create a new folder `openwakeword` so that you have `/share/openwakeword`.
+3. Drop your shiny new wake word model file (`.tflite`) into that folder.
+4. Restart the **Wyoming** integration.
+   - Go to {% my integrations title="**Settings** > **Devices & Services**" %} and select the **Wyoming** integration.
+   - Select the three dots menu and select **Reload**.
```

Comment:
This will not be necessary once https://github.com/home-assistant/core/pull/101827 ships. (which we aim to do before the release tomorrow)

```suggestion
```